src/main/java/com/exasol/errorcodecrawlermavenplugin/ErrorCodeCrawlerMojo.java

```diff
@@ -42,6 +44,23 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         log.info("Found " + crawlResult.getErrorMessageDeclarations().size() + " valid error message declarations.");
     }
 
+    private int getJavaLanguageLevel() {
```

Comment:
Somehow it got me a bit confused to understand what "JavaLanguageLevel" meant.
Perhaps we can change to "JavaMajorRelease", or "JavaVersion" or something related?